### ISSUE TITLE
Missing curve in roc_plot

### ISSUE THREAD

**Issue by humburg** [ambiata/classifierplots#19]
I've encountered an issue (using version 1.4.0) where the ROC curve is missing from the plot produced by *roc_plot()*. See the image below for an example. As far as I can tell no errors or warnings are produced in the process. 
![roc_plot](https://user-images.githubusercontent.com/3471240/100288772-86b7ef80-2fcb-11eb-9820-7e8ed5157cdb.png)

**Comment by Juke34** [comment]
Same issue here
R 4.1.1
classifierplots 1.4.0

**Comment by adefazio** [comment]
Hi, does the small test example work for you? 
```roc_plot(example_predictions$test.y, example_predictions$pred.prob)```
I just did a fresh install of R and ran this example, it works for me.

If it is also working for you, I will need more information to debug the problem.

**Comment by humburg** [comment]
Yes, that does work. What sort of additional information would you like?

**Comment by adefazio** [comment]
If your data is sharable that it easiest. If not, can you check for nan, inf etc. in your data. If there are a lot of duplicate values in the pred.prob values could cause issues. Do you have any issues running other ROC curve plotting libraries? 
A percentile plot of test.y and pred.prob could help diagnose things. Is pred.prob in the range [0,1]?

**Comment by humburg** [comment]
This isn't my data but I see what I can do to get you more details.
